importers/exchanges/binance.js

```diff
@@ -24,13 +24,13 @@ var fetch = () => {
 var handleFetch = (unk, trades) => {
   if (trades.length > 0) {
     var last = moment.unix(_.last(trades).date).utc();
-    var next = last.clone();
+    var next = from.clone().add(1, 'h').subtract(1, 's');
   } else {
```

Comment:
I'm curious why you need to subtract 1 second here. Could you comment about that?

Reply:
Binance has millisecond resolution in their queries, Gekko only deals with seconds in the imported trades. If you did not subtract 1 second then you potentiall lose 999 milliseconds of records on every iteration.

Reply:
Ah I see. Binance limits queries to 3600 seconds (1 hour). But if you just specify start and end ranges in the naive way, you're actually asking for 3601 seconds, and Binance will truncate it? Hence you lose 999 ms of data.
Makes sense to me. Ship it!

Reply:
@thecodeboss In re-reading this... it's an error. Since I moved away from using trade times, and to fixing the iterator, the minus 1 second is no longer necessary and won't loose any data. I'll remove it :)